packages/app-store/closecom/components/EventTypeAppCardInterface.tsx

```diff
@@ -0,0 +1,32 @@
+import { usePathname } from "next/navigation";
+
+import AppCard from "@calcom/app-store/_components/AppCard";
+import useIsAppEnabled from "@calcom/app-store/_utils/useIsAppEnabled";
+import type { EventTypeAppCardComponent } from "@calcom/app-store/types";
+import { WEBAPP_URL } from "@calcom/lib/constants";
+
+const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({ app, eventType }) {
+  const pathname = usePathname();
+
+  const { enabled, updateEnabled } = useIsAppEnabled(app);
+
+  // CRM backwards compatibility
+  if (enabled === undefined) {
+    updateEnabled(true);
+  }
```

Comment:
This will trigger 2 renders on component mount. Is there a way tell the `useIsAppEnabled` hook to default to `true` if enabled is undefined?

Reply:
I think i have resolved this issue @zomars - made changes to the useIsAppEnabled hook. Please take a look at the latest commit